CMakeLists.txt

```diff
@@ -4,7 +4,7 @@
 cmake_minimum_required(VERSION 2.8.7)
 project(SoapyFCDPP CXX C)
 
-find_package(SoapySDR "0.7.0" NO_MODULE)
+find_package(SoapySDR "0.6.0...0.7.0" NO_MODULE)
```

Comment:
The changes in the PR make it look like soapysdr 0.6 is supported with the ifdefs for the converter registry. I think my concern with this VERSION range is it will exclude the master branch "0.8.0" which should be fine to compile for. Perhaps the version should be 0.6 for version 0.6 and above supported.

Reply:
Yep, I wanted to specify 0.6.0+ but CMake cannot do that I think? Happy to put larger values in.

Reply:
I'm not super familiar with the syntax, but a single version number with no markup implies that cmake will look for that version or higher. so "0.6.0" means look for `version >= 0.6.0` Most of the projects just specify the minimum version number like this and it seems to be fine.

Reply:
OK, I'll just pop that commit out.